### PR TITLE
Fix test case issues for BVT category

### PR DESCRIPTION
**Pull request by LiliDeng**
1 ClearOS the boot config path is different
2 no wget tool in oracle distro by default
3 add entry for more distro in timeclock cases
4 Password policy check before it run without sudo, currently framework using sudo, case will fail, update code according, and add one more situation when password contains part of username.

### CODE REVIEW

File: Testscripts/Linux/BVT-NET-IFUP-IFDOWN.sh [hunk 1]
```diff
@@ -9,6 +9,9 @@ LoopCount=10
 
 PingCheck()
 {
+    if [[ $(detect_linux_distribution) == oracle ]]; then
+        yum install -y wget
```
Comment (lizzha) [inline Testscripts/Linux/BVT-NET-IFUP-IFDOWN.sh]: no need to detect the distro, we cannot assume other distros has wget preinstalled.
just run: install_package wget

**Comment by lizzha** [comment]
also please post the test results

File: Testscripts/Linux/BVT-NET-IFUP-IFDOWN.sh (hunk 1 above)
Comment (LiliDeng) [reply Testscripts/Linux/BVT-NET-IFUP-IFDOWN.sh]: make sense, will update and post results later, thanks for review

**Comment by LiliDeng** [comment]
image

**Pull request merged**